sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -600,6 +600,9 @@ associated with `syclContext`.
 * Throws synchronously with error code `invalid` if `syclDevice`
   <<device-info-query, reports this extension as unsupported>>.
 
+* Throws synchronously with error code `invalid` if the `syclDevice` backend 
+is not supported.
+
```

Comment:
```suggestion
* Throws synchronously with error code `invalid` if the backend associated
with `syclDevice` is not supported.

```
Think this is a bit more in keeping with surrounding and general SYCL spec language.